core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -127,12 +141,14 @@ public void draw(SpriteBatch batch, float runtime) {
 				}
 			}
 			/* Does the ball hit the edge? */
-			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN && Math.toDegrees(ball.getAngle()) > 90) {
+			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN
```

Comment:
Nice. Does it changes the code and make it follow the style automatically too? or only points to where changes should be done?